app/javascript/components/settings/theme-preference-form/utils.ts

```diff
@@ -16,7 +16,7 @@ export function setThemeClassName(theme: string): void {
   }
 }
 
-function grabCurrentTheme():
+export function grabCurrentTheme():
```

Comment:
TODO: remove this fn as it is deprecated and replaced by `getCurrentTheme` from `hooks/use-theme-observer.tsx`